reconcile/terraform_resources.py

```diff
@@ -549,15 +549,15 @@ def cleanup_and_exit(tf=None, status=False, working_dirs=None):
     sys.exit(status)
 
 
-def write_outputs_to_vault(vault_path, ri):
+def write_outputs_to_vault(vault_path: str, resoure_specs: TerraformResourceSpecDict) -> None:
     integration_name = QONTRACT_INTEGRATION.replace('_', '-')
-    vault_client = VaultClient()
-    for cluster, namespace, _, data in ri:
-        for name, d_item in data['desired'].items():
-            secret_path = \
-                f"{vault_path}/{integration_name}/{cluster}/{namespace}/{name}"
-            secret = {'path': secret_path, 'data': d_item.body['data']}
-            vault_client.write(secret)
+    vault_client = cast(_VaultClient, VaultClient())
```

Comment:
What is happening here?

Reply:
this is mostly just hopping through some hoops to make mypy happy type wise